about.html

```diff
@@ -0,0 +1,151 @@
+    </div>
+  </section>
+
+  <section id="logo" class="logo-section container">
+    <div class="row justify-content-md-center">
+      <div class="col"></div>
+      <div class="col-11 about-logo">
+      <h1 class="about-logo__title">Cosmopolitan Affirming Community <br> Global Summit 2023 Logo</h1>
+      <div class="text-separator"></div>
+      <p class="about-logo__bio">
+        The logo of CAC Global Summit 2023 was decided through the logo competition from 8<sup>th</sup>, June to 7<sup>th</sup>, July.
+        After online public voting and Selection Committee's judjing process the logo from Malcom Charles won the prize.
+        After consultation with Pastor David we optimized the prize winning logo for this year's summit as following:
+      </p>
+      <div class="about-logo__images">
+        <img src="./elements/logo.png" alt="Logo image">
+        <h4 footer-title> <strong>C. A. C<br>Global Summit 2023 </strong></h4>
+      </div>
+      </div>
+      <div class="col"></div>
+    </div>
+  </section>
+
+  <section id="summits" class="past-summits-section">
+    <div class="row justify-content-md-center">
+      <div class="col"></div>
+      <div class="col-11 summit-section">
+      <h1 class="summit-section__title">See the past CAC Global Summits</h1>
+      <div class="text-separator"></div>
+      <p class="summit-section__bio">
+        Take a look at the last two CAC Global Summits which took place in Buenos Aires and in Warsaw respectively.
+      </p>
+      <div class="summit-section__images">
+        <img src="./elements/pstsmt1.jpeg" class="summit-section__img" alt="2021 Summit">
+        <img src="./elements/pstsmt2.jpeg" class="summit-section__img" alt="2022 Summit">
+      </div>
+      </div>
+      <div class="col"></div>
+    </div>
+  </section>
+
+<!-- Footer section -->
+  <section id="abt-footer">
+    <div class="abt-partners footer-partners">
+      <div class="footer-partners__intro">
+        <h2 class="footer-partners__title">Partners</h2>
+        <div class="text-separator"></div>
+      </div>
+      <div class="footer-partners__logo">
+        <img src="./elements/frtggle.png" alt="" class="partner-logos">
+        <img src="./elements/ftrarbnb.png" alt="" class="partner-logos">
+        <img src="./elements/ftrfrfx.png" alt="" class="partner-logos">
+        <img src="./elements/ftrgoogle.png" alt="" class="partner-logos">
+      </div>
+    </div>
+    <div class="footer-copyright">
+      <div class="footer-logo">
+        <img src="./elements/logo.png" alt="Logo image">
+        <h4 footer-title> <strong>C. A. C<br>Global Summit 2023 </strong></h4>
+      </div>
+      <div class="footer-detail">
+        <h5 class="footer-title">2023 Cosmopolitan Affirming Community</h5>
+        <p class="footer-detail__text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ratione nemo labore excepturi totam, animi optio sed reiciendis qui saepe error eius officiis?</p>
+      </div>
+    </div>
```

Comment:
- [ ] Good job styling the footer of the About page :100:, but I think it would be better if you use a dark background to match with the template provided :+1: 

**Your design**

![Screenshot from 2023-02-09 06-17-44](https://user-images.githubusercontent.com/105117832/217798714-ffed34d3-025d-47cd-a273-ff985b4edcc5.png)

**Template**

![Screenshot from 2023-02-07 08-10-00](https://user-images.githubusercontent.com/105117832/217798790-9c9e72e1-1916-47f4-9573-3445bdcc3ab7.png)